sdk/core/Azure.Core/src/ExponentialDelayStrategyInternal.cs

```diff
@@ -5,11 +5,11 @@
 
 namespace Azure.Core
 {
-    internal class ExponentialDelay : Delay
+    internal class ExponentialDelayStrategyInternal : DelayStrategy
```

Comment:
Curious: why suffix the name with `Internal`?

Reply:
There is a whole mess with the internal delay strategy types used for LROs - this is a temporary rename until we can fix the LROs once the public DelayStrategy type is shipped.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/35437

Reply:
Discussed offline.